plugins/experimental/parent_select/consistenthash.cc

```diff
@@ -297,6 +320,8 @@ PLNextHopConsistentHash::next(TSHttpTxn txnp, void *strategyTxn, const char *exc
 
   PL_NH_Debug(PL_NH_DEBUG_TAG, "nextParent NH plugin calling");
 
+  uint32_t const NO_RING_USE_POST_REMAP = uint32_t(0) - 1;
```

Comment:
Constructor-style casts are not Leif-legal but hopefully he won't see it.

Reply:
I didn't write this, this PR is just porting the existing code/changes in core to the plugin
https://github.com/apache/trafficserver/blob/3e8f2eba/proxy/http/remap/NextHopConsistentHash.cc#L235